runtime/gc_modron_standard/RootScannerReadBarrierVerifier.hpp

```diff
@@ -51,6 +51,9 @@ class MM_RootScannerReadBarrierVerifier : public MM_RootScanner
 		virtual void scanClassStaticSlots(MM_EnvironmentBase *env);
 		virtual void doClassStaticSlots(omrobjectptr_t *slotPtr);
 
+		virtual void scanDefaultValues(MM_EnvironmentBase *env);
+		virtual void doDefaultValues(omrobjectptr_t *slotPtr);
```

Comment:
It does not seem to be needed to add these to the Verifier. It's only needed if these class slots are part of indirect Remembered Set (that will at some point become concurrently scanned in Concurrent Scavenger).

In other words, this would be needed if you had added new slots to MM_ScavengerDelegate::scavengeIndirectObjectSlots() (and similar), which you have not (which, at this point, I have no clue if it's correct or not)

Reply:
Thanks! Should I just remove these methods? @amicic @tajila

Reply:
Yes. At another look, I'm even more confident this is not needed. Both Verifier hpp/cpp should be intact.